core/src/main/java/com/datastrato/gravitino/connector/authorization/RoleAuthorizationPlugin.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.datastrato.gravitino.connector.authorization;
+
+import com.datastrato.gravitino.authorization.Role;
+import com.datastrato.gravitino.authorization.RoleChange;
+import com.datastrato.gravitino.authorization.SecurableObject;
+import java.util.List;
+
+/** Interface for authorization Role plugin operation of the underlying access control system */
+interface RoleAuthorizationPlugin {
+  /**
+   * Creates a new Role into the underlying access control system.
+   *
+   * @param role The entity of the Role.
+   * @param securableObjects The securable objects of the Role.
+   * @return True if the create operation success; False if the create operation failed.
+   * @throws RuntimeException If creating the Role encounters storage issues.
+   */
+  Boolean onCreateRole(Role role, List<SecurableObject> securableObjects) throws RuntimeException;
```

Comment:
I would suggest rename the method to `onRoleXXXed`, for example, like `onRoleCreated`, `onRoleChecked`, `onUserAdded`. This will make developers better understand when this method will be invoked, it is the behavior of "after".

Reply:
DONE